bridges/snowbridge/pallets/outbound-queue-v2/src/process_message_impl.rs

```diff
@@ -18,6 +18,11 @@ impl<T: Config> ProcessMessage for Pallet<T> {
 	) -> Result<bool, ProcessMessageError> {
 		let weight = T::WeightInfo::do_process_message();
 		if meter.try_consume(weight).is_err() {
+			Self::deposit_event(Event::MessageRejected {
+				id: None,
+				payload: Some(message.to_vec()),
+				error: ProcessMessageError::Overweight(weight),
+			});
```

Comment:
see convo here https://github.com/paritytech/polkadot-sdk/pull/8015#discussion_r2013688361

Reply:
Changed in https://github.com/paritytech/polkadot-sdk/pull/8020/commits/ed489d1c5a0e6cd222c5e3efec87268ccc13a632.